source/index.html.md

```diff
@@ -639,10 +569,102 @@ Name | Type | Description
         on_close: () => {
           console.log('on_close!');
         }
-      });
-      highlineLink.open();
-    }
+    });
+    highlineLink.open();
   </script>
 </body>
 </html>
+```
+
+
+## Building the payload
+### One-Step Payment Connection
+```javascript
+POST highline_link/encode
+Basic Auth
+Request:
+{
+ "enrollment_id",
+ "feature": "direct_deposit_payment"
+}
+ 
+Response:
+{
+ "payload": "encrypted_value"
+}
+
+```
+
+### Two-Step Payment Connection
+
+
+```javascript
+// Step 1
+POST highline_link/encode
+Basic Auth
+Request:
+{
+ "feature": "direct_deposit_payment"
+}
+ 
+Response:
+{
+ "payload": "encrypted_value"
+}
+
+
+// Step 2
+POST highline_link/encode
+Basic Auth
+Request:
+{
+ "enrollment_id",
+ "feature": "direct_deposit_payment"
+}
+ 
+Response:
+{
+ "payload": "encrypted_value"
+}
+
+```
+
+
+### Employment Data Connection
+```javascript
+POST highline_link/encode
+Basic Auth
```

Comment:
Once we explain, how authentication works in the authentication section of the doc, we just say
Bearer AccessToken